cadquery/occ_impl/exporters/svg.py

```diff
@@ -128,7 +129,7 @@ def getSVG(shape, opts=None):
         Export a shape to SVG
     """
```

Comment:
I personally need docs on these opts here, particularly a reference for `projectionDir`. Something like
> a `projectionDir` of (-1, 1, 1) displays the model in an isometric view, with the z-axis pointing upwards on the page, x to the right and y to the left

I'll forget every time if this direction points from the camera to the model or vice versa and have to check the doc string. :sweat_smile:

Reply:
I'm actually hoping to write a new import/export section of the docs that expands on the current references. I can update the doc string of getSVG too.

Reply:
Added a docstring and #598 to keep track of this.